rustls-post-quantum/src/lib.rs

```diff
@@ -4,11 +4,11 @@
 //! post-quantum-secure key exchange algorithm.
 //!
 //! X25519MLKEM768 is pre-standardization, so you should treat
-//! this as experimental.  You may see unexpected interop failures, and
-//! the algorithm implemented here may not be the one that eventually
-//! becomes widely deployed.
+//! this as experimental.  You may see unexpected interop failures -- please
+//! report these.  X25519MLKEM768 is becoming widely deployed, eg, by
```

Comment:
Maybe worth being a touch more specific about where/how to report interop issues?

Something like: "You may see unexpected interop failures -- please report these as GitHub issues on the Rustls repository, and the issue tracker associated with the peer software" ? 

or just "You may see unexpected interop failures -- please report these as GitHub issues on the Rustls repository"?

Reply:
altered this to:

> X25519MLKEM768 is pre-standardization, so you should treat this as experimental. You may see unexpected connection failures (such as [tldr.fail](https://tldr.fail/)) – [please report these to us](https://github.com/rustls/rustls/issues/new?assignees=&labels=&projects=&template=bug_report.md&title=). X25519MLKEM768 is becoming widely deployed, eg, by [Chrome](https://security.googleblog.com/2024/09/a-new-path-for-kyber-on-web.html) and [Cloudflare](https://blog.cloudflare.com/pq-2024/#ml-kem-768-and-x25519).

Reply:
Much better, thank you :rocket: